docs/events.rst

```diff
@@ -74,7 +74,7 @@ as long as you don't absolutely need it.
 
 
 Event: ``on_start``
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Why you added more `^` symbols?

Reply:
Idk, I think I didn't understand yet the "rule" that it doesn't matter how many characters there are, as long as it's more than the length of the title